Gems/PhysX/Code/Tests/PhysXJointsTest.cpp

```diff
@@ -209,6 +209,60 @@ namespace PhysX
         EXPECT_TRUE(followerEndPosition.GetZ() > followerPosition.GetZ());
     }
 
+    TEST_F(PhysXJointsTest, Joint_BallJoint_GlobalConstraint)
+    {
+        // Place lead on top of follower, tie them together with ball joint and send the follower moving sideways in the X and Y directions.
+        // The follower should swing up
+
+        const AZ::Vector3 followerPosition(0.0f, 0.0f, -1.0f);
+        const AZ::Vector3 followerInitialLinearVelocity(10.0f, 10.0f, 0.0f);
+
+        const AZ::Vector3 jointLocalPosition(0.0f, 0.0f, 2.0f);
+        const AZ::Quaternion jointLocalRotation = AZ::Quaternion::CreateRotationY(90.0f);
+        const AZ::Transform jointLocalTransform = AZ::Transform::CreateFromQuaternionAndTranslation(jointLocalRotation, jointLocalPosition);
+
+        //do not set the lead entity as that makes this a global constraint
+        auto jointConfig = AZStd::make_shared<JointComponentConfiguration>();
+        jointConfig->m_localTransformFromFollower = jointLocalTransform;
+
+        auto jointLimits = AZStd::make_shared<JointLimitProperties>();
+        jointLimits->m_isLimited = false;
+
+        auto followerEntity = AddBodyColliderEntity<BallJointComponent>(
+            m_testSceneHandle, followerPosition, followerInitialLinearVelocity, jointConfig, nullptr, jointLimits);
+
+        const AZ::Vector3 followerEndPosition = RunJointTest(m_defaultScene, followerEntity->GetId());
+
+        EXPECT_TRUE(followerEndPosition.GetZ() > followerPosition.GetZ());
```

Comment:
`EXPECT_GT` could be used instead and will provide more information on failure.

Same in the other unit test.